examples/mobile/UIComponents/components/appbar/multiline/multiline-title.html

```diff
@@ -12,6 +12,9 @@
 <body>
 	<div class="ui-page" id="multilineTitle">
 		<header>
+			<div class="ui-appbar-left-icons-container">
```

Comment:
we need to decide and be consistent if we add back button to submenus or examples too.
This is example not submenu.
If we add back button to examples we will need to add to all of them
e.g. also
examples/mobile/UIComponents/components/etc/units/units.html
examples/mobile/UIComponents/components/etc/units/fonts.html
examples/mobile/UIComponents/components/etc/interactive/coverflow/index.html
examples/mobile/UIComponents/components/etc/interactive/interactive3D/index.html
examples/mobile/UIComponents/components/etc/about/index.html